features/support/object_model_pages/shop/employee/registration/ee_family_information.rb

```diff
@@ -166,4 +170,8 @@ def self.dependent_add_mailing_address_btn
   def self.confirm_member_btn
     'span[class="btn btn-primary btn-br pull-right mz"]'
   end
+
+  def self.lives_with_primary
+    'input[id="dependent_same_with_primary'
```

Comment:
let's use a `data-cuke="something"` selector for all new page object model selectors